modules/dnn/src/dnn.cpp

```diff
@@ -1262,7 +1266,8 @@ struct Net::Impl
             allocateLayer(lid, layersShapes);
         }
 
-        fuseLayers(blobsToKeep_);
+        if (0)
+            fuseLayers(blobsToKeep_);
```

Comment:
this change will seriously affect performance of CPU branch. By "seriously" I mean 50-100%. If you want to disable fusion in the case of OpenCL target, please, do so.